From 80d8289fb554704ed21000baaf334d549641804d Mon Sep 17 00:00:00 2001
From: Max Reitz <mreitz@redhat.com>
Date: Sun, 19 Jan 2014 18:07:59 +0100
Subject: [PATCH 25/34] qapi-schema: Update description for NewImageMode

RH-Author: Max Reitz <mreitz@redhat.com>
Message-id: <1390154881-17140-6-git-send-email-mreitz@redhat.com>
Patchwork-id: 56818
O-Subject: [RHEL-7.0 qemu-kvm PATCH v2 5/7] qapi-schema: Update description for NewImageMode
Bugzilla: 921890
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>

BZ: 921890

If the NewImageMode is "absolute-paths" but no backing file is available
(e.g., when mirroring a device with an unbacked image), the target image
will not be backed either. This patch updates the documentation in
qapi-schema.json accordingly.

Signed-off-by: Max Reitz <mreitz@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Reviewed-by: Benoit Canet <benoit@irqsave.net>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit 1296c2f0bc8283164c925bf587e497511de7f569)

Signed-off-by: Max Reitz <mreitz@redhat.com>
---
 qapi-schema.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qapi-schema.json |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/qapi-schema.json b/qapi-schema.json
index a783e8e..6158ea5 100644
--- a/qapi-schema.json
+++ b/qapi-schema.json
@@ -1722,7 +1722,8 @@
 # @existing: QEMU should look for an existing image file.
 #
 # @absolute-paths: QEMU should create a new image with absolute paths
-# for the backing file.
+# for the backing file. If there is no backing file available, the new
+# image will not be backed either.
 #
 # Since: 1.1
 ##
-- 
1.7.1

